src/components/API.js

```diff
@@ -0,0 +1,152 @@
+export default class API {
+  constructor(options) {
+    this._options = options;
+    this._baseURL = options.baseUrl;
+    this._authtoken = options.headers.authorization;
+    this._contentTypeJSON = options.headers["Content-Type"];
+  }
+
+  getInitalCards() {
+    return fetch(`${this._baseURL}/cards`, {
+      method: "GET",
+      headers: {
+        authorization: this._authtoken,
+      },
+    })
+      .then((res) => {
+        if (res.ok) {
+          return res.json();
+        }
+        return Promise.reject(res);
+      })
+      .catch((err) => {
+        console.error(`Error Loading Cards: ${err.status}`);
```

Comment:
It’s better not to use `catch` in `Api` because you need to `catch` possible errors at the very **end** of any request and this end is in `index.js`   after all blocks `then`. And in real projects blocks `catch` usually open error popups and so on.